### PR TITLE
Add table reporting on Docker builds.

### PR DESCRIPTION
**Pull request by vincentmr**
[sc-55271]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,19 +39,24 @@ All entries in the matrix are tested against PennyLane latest (GitHub master).
...
 | :---------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [`build-branch-dev`](https://github.com/PennyLaneAI/qml/blob/master/.github/workflows/build-branch-dev.yml)       | [![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/PennyLaneAI/qml/build-branch-dev.yml)](https://github.com/PennyLaneAI/qml/actions/workflows/build-branch-dev.yml)       |
 | [`build-branch-master`](https://github.com/PennyLaneAI/qml/blob/master/.github/workflows/build-branch-master.yml) | [![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/PennyLaneAI/qml/build-branch-master.yml)](https://github.com/PennyLaneAI/qml/actions/workflows/build-branch-master.yml) |
 
+## Docker builds
```
Comment (timmysilv) [inline README.md:53]:
```suggestion
## Lightning Docker builds
```

**Review by timmysilv — APPROVED** [review]
one comment to avoid ambiguity, but otherwise looks good!

**Comment by vincentmr** [comment]
@timmysilv Thanks! I was thinking other Docker workflows could be reported in the table eventually, but let's keep Lightning in the title to avoid ambiguity for now.

**Comment by trbromley** [comment]
How often are the actions run?

**Comment by vincentmr** [comment]
> How often are the actions run?

Every day of the week @4am.

**Comment by vincentmr** [comment]
> are the jobs supposed to be running? I'm seeing "no status" on the master badge

That's because there is a bit of a chicken-and-egg situation between the repos. The jobs have not run on `master` yet, but the labels showed correctly when `master` was replaced by the correct branch name. That said, it's been merged now, so I'll trigger the workflows and let's see if everything shows correctly before merging.

**Comment by vincentmr** [comment]
Both labels should show green now, I think we're good to go.

**Review by timmysilv — APPROVED** [review]

**Review by lillian542 — APPROVED** [review]

**Comment by Alex-Preciado** [comment]
Merging on behalf of @vincentmr 🙏🏼

**Pull request merged**